docs/source/user/settings.rst

```diff
@@ -882,3 +882,19 @@ The max number of SMS tokens a single user can request within a day.
 
 The max number of tokens which can be requested from the same IP address
 during the same day.
+
+``OPENWISP_RADIUS_SMS_MESSAGE_TEMPLATE``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Default**: ``{organization_name} verification code: {code}``
```

Comment:
You changed the string and broke the translations.

```
from django.utils.translation import gettext as _
from django.utils.translation import activate, get_language_info
activate("it")
_(org.radius_settings.sms_message)
 '{organization_name} verification code: {code}'
```

Why did you change `{organization}` to `{organization_name}`?
**That is a backward incompatible change.**